CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## [1.3.1]
```

Comment:
We need to keep this as `[Unreleased]` for the `release-prepare` script to work, IIRC

Reply:
Oh, do we have to make another pull request to adjust the changelog version afterward then?